core/src/main/scala/com/yahoo/maha/core/RequestModel.scala

```diff
@@ -446,6 +446,7 @@ object RequestModel extends Logging {
               if(publicFact.dimToRevisionMap.isEmpty) Map.empty
               else {
                 requestedAliasList
+                    .filter(publicFact.dimToRevisionMap.keys.toSet.contains(_))
```

Comment:
it means dimToRevisionMap is compulsory map to be filled in every cube ? 
Should not be the case

Reply:
This only filters input requested cols so that it only checks the ones requested in the map.